src/index.ts

```diff
@@ -1561,7 +1560,7 @@ function FlatpickrInstance(
       )
         // disabled by date
         return bool;
-      else if (typeof d === "string" && dateToCheck !== undefined) {
```

Comment:
This check is removed because we already have this on line 1543 `if (dateToCheck === undefined) return false;`